machine_learning_hep/data/data_run3/database_variations_LcJet_pp_jet_obs.yml

```diff
@@ -105,7 +216,7 @@ categories:
                                       model:
                                         fn: 'SUM::mctot(mcfrac[0.,1.]*sig, mcbkg)'
                                   - ptrange: [1., 5.]
-                                    range: [2.13, 2.44]
+                                    range: [2.18, 2.40]
```

Comment:
Shouldn't we vary only the lower bound for massmin?

Reply:
At the moment we are not varying the fit range for the Lc. Also why would you want to change only the lowe bound, in case? I just wanted to update the values with the new nominal ranges in the lower pt bins but I reported the wrong value for the minimum (it should be 2.16)